### PR TITLE
Improve selection of cluster

### LINKED ISSUE #62: Improve selection of cluster
Cluster selection should be improved:

 * add documentation about the goal of cluster selection.
 * show in the kafka explorer which cluster is selcted
 * provide the Unselect command.

### PR DESCRIPTION
**Pull request by angelozerr**
Improve selection of cluster

Fixes #62

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
My work is not finished, but it starts working:

ExporerSelectionDemo

File: src/constants.ts [hunk 1]
```diff
@@ -48,6 +48,11 @@ export class Icons {
     }
 }
 
+
+export enum GlyphChars {
+    Check = '\u2713'
```
Comment (angelozerr) [inline src/constants.ts:53]: Just for information, the idea of GlyphChars comes from the vscode-gitlens https://github.com/eamodio/vscode-gitlens/blob/7f549c66e7ade7c569f5acd906d170646d7ab5d9/src/views/nodes/branchNode.ts#L392

**Comment by angelozerr** [comment]
The PR can be reviewed, it shows the selected cluster is the tree and manage too the enable/disable of `Select Cluster` contextual menu according to if cluster is already selected or not.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ All notable changes to Kafka extension will be documented in this file.
 - Internal topics are now hidden by default. See [#29](https://github.com/jlandersen/vscode-kafka/issues/29) and [#74](https://github.com/jlandersen/vscode-kafka/pull/74).
 - Elements are now sorted alphabetically in the Kafka explorer. See [#63](https://github.com/jlandersen/vscode-kafka/issues/63).
 - Clusters are now sorted in the cluster selection wizard. See [#83](https://github.com/jlandersen/vscode-kafka/issues/83).
+- Selection of cluster is now visible in the explorer and `Select Cluster` menu item is displayed only for unselected cluster. See [#82](https://github.com/jlandersen/vscode-kafka/issues/82).
```
Comment (fbricon) [inline CHANGELOG.md:19]: Cluster selection is now ... the `Select Cluster` menu is displayed only for unselected clusters.

Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ All notable changes to Kafka extension will be documented in this file.
 - Internal topics are now hidden by default. See [#29](https://github.com/jlandersen/vscode-kafka/issues/29) and [#74](https://github.com/jlandersen/vscode-kafka/pull/74).
 - Elements are now sorted alphabetically in the Kafka explorer. See [#63](https://github.com/jlandersen/vscode-kafka/issues/63).
 - Clusters are now sorted in the cluster selection wizard. See [#83](https://github.com/jlandersen/vscode-kafka/issues/83).
+- Selection of cluster is now visible in the explorer and the `Select Cluster` menu is displayed only for unselected clusters. See [#82](https://github.com/jlandersen/vscode-kafka/issues/82).
```
Comment (fbricon) [inline CHANGELOG.md:19]: Currently selected cluster

**Review by fbricon — APPROVED** [review]
Apart from a small change in the changelog, the rest LGTM. Feel free to merge once you fixed the changelog

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

**Comment by angelozerr** [comment]
Thanks @fbricon for your review.

**Pull request merged**